src/main/java/org/jitsi/videobridge/cc/GenericAdaptiveTrackProjectionContext.java

```diff
@@ -232,13 +234,13 @@ else if (payloadType instanceof Vp9PayloadType)
             long destinationTimestamp
                 = computeDestinationTimestamp(rtpPacket.getTimestamp());
 
-            if (RTPUtils.isOlderSequenceNumberThan(
+            if (RtpUtils.Companion.isOlderSequenceNumberThan(
```

Comment:
Do we maybe want to annotate the RtpUtils functions with @JvmStatic so we don't have to say .Companion in Java?